ccm_web/server/src/canvas/canvas.service.ts

```diff
@@ -18,7 +18,37 @@ import baseLogger from '../logger'
 const logger = baseLogger.child({ filePath: __filename })
 
 type SupportedAPIEndpoint = '/api/v1/' | '/api/graphql/'
-const requestorOptions: GotOptions = { retry: { limit: 2, methods: ['POST', 'GET', 'PUT', 'DELETE'] } }
+const requestorOptions: GotOptions = {
+  retry: {
+    limit: 2,
+    methods: ['POST', 'GET', 'PUT', 'DELETE'],
+    statusCodes: [403]
```

Comment:
Yes, you're right.  I had meant to go back and add in all the defaults, but I forgot.  I will try to get the list of defaults from Got and add `403` to it.  If I can't do that, then I'll hard-code their default list and include `403`.

Reply:
got@11.8.2 doesn't appear to have any `statusCodes` defined by default, which is probably why its retry feature didn't work as I expected until after I added `403`.  I'm hard-coding the list with the codes from got@12 and adding `403`.  In the future, we can replace the hard-coded list by replacing it with something like `defaultInternals.retry.statusCodes.concat(403)`.

Reply:
Thanks for figuring out this retry stuff and making sure it's working.

Reply:
Are you sure there are no status codes in 11.8.2? The docs from `npm` seem to suggest otherwise: https://www.npmjs.com/package/got/v/11.8.2#retry

Not sure if that means you can do your default thingy now, but I was confused by your comment.

Reply:
I saw the same documentation here: https://github.com/sindresorhus/got/blob/v11.8.2/readme.md#retry

However, when I browsed through the code for 11.8.2 (in the GitHub repo web UI and inspecting the version that was installed on my machine), I couldn't find any code that was setting those defaults.

I concluded the documentation must be faulty.

Reply:
@lsloan, what about here? https://github.com/sindresorhus/got/blob/v11.8.2/source/index.ts

Reply:
Wow!  OK.  I don't understand.  I was looking through the code for all mentions of `statusCodes` and some of the specific code numbers mentioned in the docs, like `429`, `524`, etc., but I didn't find it.  That's weird.

Now that I know it's there, I think I figured out why I didn't see it.  I think my IDE ignores JS files when there are TS files present, because it assumes the JS files are generated/compiled from TS files.

OK.  I'll add an issue to remove the hard-coded list.

Reply:
Opened #194.